src/component/framework/association.js

```diff
@@ -23,6 +25,14 @@ export class Association {
     if (this.manyAssociation) {
       this.manyAssociation = this.formField.model[this.formField.element.manyAssociation];
     }
+
+    if (typeof element.selectablePlaceholder !== 'undefined') {
```

Comment:
Instead of these if statements you can vind them directly to the association-select.